shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -107,18 +106,50 @@
         @Override
         public void createAndroidViewForPlatformView(
             @NonNull PlatformViewsChannel.PlatformViewCreationRequest request) {
-          // API level 19 is required for android.graphics.ImageReader.
+          // API level 19 is required for `android.graphics.ImageReader`.
           ensureValidAndroidVersion(Build.VERSION_CODES.KITKAT);
-          platformViewRequests.put(request.viewId, request);
+
+          if (request == null) {
+            throw new IllegalStateException(
```

Comment:
IllegalArgumentException?

Reply:
You right, this is an argument exception. I also realized that this isn't necessary because `request` is `NonNull`.